process/working-group-lifecycle.md

```diff
@@ -77,3 +77,15 @@ A WG is expected to continue operating per the above guidelines, and to provide
 * All significant artifacts should be archived as appropriate
 * Meeting series removed from calendar, mail list closed, and any other administrative items
   similarly put to closure as needed
+
+## Submission Process
+
+### Working Group creation or change of lifecycle stage
+
+For initiating the creation of a new WG or for requesting a change of a WG lifecycle stage, an application must be submitted to the TAC. To this end, a lead of the WG (or anyone in the case of an Archived WG) creates a PR in this repository with the following changes:
```

Comment:
I like this! If we wanted to get fancy, maybe we could guide people through this process by using a [pull request template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)? On the other hand, maybe simple is better.

Reply:
I agree. I played with templates when I first experimented with this over a year ago but we need to have several templates and have people choose the right one. At the time, this seemed rather convoluted but I'm no github expert and maybe this has now become easier. If you happen to know anyone at GitHub who might be knowledgeable about this feel free to ping him and submit a PR! :innocent: